AdyenCard/Components/Stored Card/StoredCardComponent.swift

```diff
@@ -38,17 +43,15 @@ internal final class StoredCardComponent: PaymentComponent, PresentableComponent
     
     /// :nodoc:
     internal lazy var storedCardAlertManager: StoredCardAlertManager = {
-        Analytics.sendEvent(component: paymentMethod.type, flavor: _isDropIn ? .dropin : .components, environment: environment)
+        Analytics.sendEvent(
+            component: paymentMethod.type,
+            flavor: _isDropIn ? .dropin : .components,
```

Comment:
` _isDropIn ? .dropin : .components ` this could be a helper for `Component` ? 🤔

Reply:
maybe we could do this in another PR